Tools/2MGFX/EffectObject.writer.cs

```diff
@@ -140,7 +140,7 @@ public void Write(BinaryWriter writer, Options options)
 
         private static void WriteParameters(BinaryWriter writer, d3dx_parameter[] parameters, int count)
         {
-            writer.Write((byte)count);
+            writer.Write((short) count);
```

Comment:
Should we update those two just in case?
https://github.com/Jjagg/MonoGame/blob/cb24746589835f0d4e2b7c041b6b0c8659979737/Tools/2MGFX/EffectObject.writer.cs#L159-L160